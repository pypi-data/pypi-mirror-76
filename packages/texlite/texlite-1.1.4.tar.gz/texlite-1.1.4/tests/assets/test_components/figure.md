![The Universe](graphics/universe.jpg)
![](graphics/universe.jpg)
![Not The Universe](graphics/thisfiledoesnotexist.jpg)