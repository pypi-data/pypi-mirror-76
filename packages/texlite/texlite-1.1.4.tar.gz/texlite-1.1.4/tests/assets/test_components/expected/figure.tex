% meta
\documentclass[10pt]{extarticle}

% packages
\usepackage[utf8]{inputenc}
\usepackage[margin=1.6in]{geometry}
\usepackage{hyperref}
\usepackage{amsmath}
\usepackage{amssymb}

% preamble commands
\linespread{1.0}
\usepackage{graphicx}
\graphicspath{{tests/assets/test_components/}}

% document details
\date{}

% ------------------------------------------------------------------------------
\begin{document}
% ------------------------------------------------------------------------------

\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{graphics/universe.jpg}
\caption{The Universe}\end{figure}

\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{graphics/universe.jpg}
\end{figure}

\begin{figure}[h!]
\centering
\texttt{Image Not Found}
\caption{Not The Universe}\end{figure}

% ------------------------------------------------------------------------------
\end{document}
% ------------------------------------------------------------------------------

